### ISSUE TITLE
Accessibility - Poor Color Contrast with ILAB Logo and DOL Seal

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#89]
By changing the background image on the main screen, there is not enough color contrast for the white DOL seal and ILAB logo on the tan-colored background image.

**Comment by jfc3-dol** [comment]
There is still a color contrast issues for the DOL seal and ILAB logo on the tan-colored part of the background image on the home screen, so I'm leaving the issue open.

**Comment by jfc3-dol** [comment]
@Abychalil,

Please see the attached screenshot showing the poor color contrast of the ILAB text and DOL seal at the bottom of the screen. With having a tan/light-colored background down there, the text should be black ( #000000 ) to pass.

![ilab - sweat and toil  - android - home - power color contrast of ILAB text and DOL seal in logo](https://user-images.githubusercontent.com/10423487/88325445-1f272600-ccf3-11ea-93aa-491afb6704a6.png)

**Comment by jfc3-dol** [comment]
ILAB developers/team response

This change was done in iOS but missed in android (not committed in GitHub, so final build doesn’t reflect this change). We’ll publish this change as a part of Sprint 2 deliverables.

jfc iii - response

I figured that was probably the case; the code did not get updated/posted. I'm looking forward to testing the update.

**Comment by pbhatt17** [comment]
@jfc3-dol 

This issue is fixed in latest build.

@harpreetsdevgit

**Comment by jfc3-dol** [comment]
@pbhatt17 @harpreetsdevgit

Once I get the build from the OPA Tech team, most likely early next week, I will test this again.

**Comment by jfc3-dol** [comment]
@pbhatt17 

You can hardly see the DOL seal highlighted below, so the color contrast does not pass. Please see the attached screenshot.

<img width="360" alt="ilab - sweat and toil - home page - poor color contrast of DOL seal in footer section" src="https://user-images.githubusercontent.com/10423487/129055180-78fb3865-86d2-4c1e-af75-386fb5b4b116.png">

**Comment by jfc3-dol** [comment]
@pbhatt17 

The new home page image that is attached passes the color contrast check for the DOL seal in the lower right. So I'm going to close this issue.

![Sweat and Toil - Android - home page showing image and DOL seal, etc](https://user-images.githubusercontent.com/10423487/134543684-ecd57a81-7087-4be9-9682-67343bf07d78.jpg)
